deploy/helm/sumologic/conf/logs/logs.source.containers.conf

```diff
@@ -15,6 +15,8 @@
 <label @NORMAL>
   <filter containers.**>
     @type kubernetes_metadata
+    cache_size  {{ .Values.sumologic.k8sMetadataFilter.cacheSize }}
```

Comment:
These also need to be added to the `enhance_k8s_metadata` plugin below

Reply:
Just noticed these are defined below, line 29 and line 30.
Since we're still using ENV vars until the major release, we should stay consistent with this and use the new ENV var `K8S_METADATA_FILTER_CACHE_SIZE` and `K8S_METADATA_FILTER_CACHE_TTL` in all of the plugins as well, for consistency

Reply:
Thanks Sam! Will float a PR to fix that.